pymc/smc/sample_smc.py

```diff
@@ -56,50 +56,53 @@ def sample_smc(
 
     Parameters
     ----------
-    draws: int
+    draws : int, default 2000
         The number of samples to draw from the posterior (i.e. last stage). And also the number of
         independent chains. Defaults to 2000.
-    kernel: SMC Kernel used. Defaults to pm.smc.IMH (Independent Metropolis Hastings)
-    start: dict, or array of dict
+    kernel : SMC_kernel, optional
+        SMC kernel used. Defaults to :class:`pymc.smc.smc.IMH` (Independent Metropolis Hastings)
+    start : dict or array of dict, optional
         Starting point in parameter space. It should be a list of dict with length `chains`.
         When None (default) the starting point is sampled from the prior distribution.
-    model: Model (optional if in ``with`` context)).
-    random_seed: int
-        random seed
-    chains : int
+    model : Model (optional if in ``with`` context).
+    random_seed :  int, array_like of int, RandomState or Generator, optional
```

Comment:
```suggestion
    random_seed :  int, array_like of int, RandomState or numpy_Generator, optional
```

This will render as a link as it uses one alias: https://github.com/pymc-devs/pymc/blob/main/docs/source/conf.py#L80. Otherwise, `Generator` is ambiguous for sphinx to know what it refers to